packages/extension-ui/src/Popup/Welcome.tsx

```diff
@@ -28,30 +28,42 @@ const Welcome = function ({ className }: Props): React.ReactElement<Props> {
 
   return (
     <>
-      <Header text={t<string>('Welcome')} />
       <div className={className}>
-        <p>{t<string>('Before we start, just a couple of notes regarding use:')}</p>
-        <Box>
-          <List>
-            <li>{t<string>('We do not send any clicks, pageviews or events to a central server')}</li>
-            <li>{t<string>('We do not use any trackers or analytics')}</li>
-            <li>{t<string>("We don't collect keys, addresses or any information - your information never leaves this machine")}</li>
-          </List>
-        </Box>
-        <p>{t<string>('... we are not in the information collection business (even anonymized).')}</p>
+        <img
+          className='centered'
+          src={secureIMG}
+        />
+        <p className='heading'>{t<string>('Your privacy is protected')}</p>
+        <List>
+          <li>{t<string>('We do NOT send any clicks, pageviews or events to a central server')}</li>
+          <li>{t<string>('We do NOT use any trackers or analytics')}</li>
+          <li>{t<string>('We do NOT collect keys, addresses or any information - your information never leaves this machine')}</li>
+          <li>{t<string>('We are NOT in the information collection business (even anonymized).')}</li>
+        </List>
+
       </div>
-      <VerticalSpace />
       <ButtonArea>
-        <Button onClick={_onClick}>{t<string>('Understood, let me continue')}</Button>
+        <Button onClick={_onClick}>{t<string>('Got it!')}</Button>
       </ButtonArea>
     </>
   );
 };
 
 export default styled(Welcome)(({ theme }: Props) => `
-  p {
-    color: ${theme.subTextColor};
-    margin-bottom: 6px;
-    margin-top: 0;
+ 
+  .centered{
```

Comment:
Could you add spaces between class names and open bracket? And that empty line on top isn't really needed :) 

BTW: do you have prettier connected? I see the repo has prettier config so I would think it should correct some of those things on save automatically if you IDE is configured to use it 🤔